apps/hash-ai-worker-ts/src/activities/flow-activities/shared/propose-entities-from-facts/propose-entity-from-facts.ts

```diff
@@ -137,15 +137,17 @@ const generatePropertyMetadata = (params: {
       );
     }
 
-    propertyMetadata.push({
-      path: [baseUrl],
+    if (!propertyMetadata.value) {
+      propertyMetadata.value = {};
+    }
```

Comment:
[nullish assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment)
```suggestion
    propertyMetadata.value ??= {};
```